3. Installation/3. Docker Containers/systemd/README.md

```diff
@@ -4,20 +4,101 @@ Deploy [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) to Linux that ru
 
 ## Prerequisite
 
-You need to have [docker](https://docs.docker.com/linux/started/) and [docker-compose](http://docs.docker.com/compose/) installed.
+You need to have [docker](https://docs.docker.com/linux/started/) installed.
 
 ## How to run Rocket.Chat on systemd.
 
+Create three systemd files. One for the database, one for the database replica and one for rocketchat.
+
+mongo.service:
+```
+[Unit]
+Description=mongo
+Requires=docker.service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=dockeruser
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill mongo
```

Comment:
Have you been using this in production?  This kind of makes me nervous.  Actually blowing away the mongo container every time.  I know below you have volume mounts... But seems like you would want to make sure your image stayed exactly the same.